sky/catalog/data_fetchers/fetch_nebius.py

```diff
@@ -0,0 +1,308 @@
+                calculator_service.estimate_batch(price_request,
+                                                  timeout=TIMEOUT),
+            ))
+
+    # wait all futures to complete and collect results
+    result = []
+    for platform, preset, future in futures:
+        platform_name = platform.metadata.name
+        result.append(
+            PresetInfo(
+                region=region,
+                fullname=f'{platform_name}_{preset.name}',
+                name=preset.name,
+                platform_name=platform_name,
+                gpu=preset.resources.gpu_count or 0,
+                vcpu=preset.resources.vcpu_count,
+                memory_gib=preset.resources.memory_gibibytes,
+                accelerator_manufacturer=ACCELERATOR_MANUFACTURER
+                if platform_name.startswith('gpu-') else '',
+                accelerator_name=platform_name.split('-')[1].upper()
+                if platform_name.startswith('gpu-') else '',
+                price_hourly=decimal.Decimal(
+                    future.wait().hourly_cost.general.total.cost),
+            ))
+
+    return result
+
+
+def _write_preset_prices(presets: List[PresetInfo], output_file: str) -> None:
+    """Writes the provided preset information to a CSV file.
+
+    Args:
+        presets (List[PresetInfo]): A list of PresetInfo objects to write.
+        output_file (str): The path to the output CSV file.
+    """
+    # Set up the CSV writer to output to stdout
+    with open(output_file, 'w', encoding='utf-8') as out:
+        header = [
+            'InstanceType',
+            'AcceleratorName',
+            'AcceleratorCount',
+            'vCPUs',
+            'MemoryGiB',
+            'Price',
+            'Region',
+            'GpuInfo',
+            'SpotPrice',
+        ]
+        writer = csv.DictWriter(out, fieldnames=header)
+        writer.writeheader()
+
+        for preset in sorted(presets,
+                             key=lambda x:
+                             (bool(x.gpu), x.region, x.platform_name, x.vcpu)):
+            gpu_info = ''
+            if preset.gpu > 0:
+                gpu_info_dict = {
+                    'Gpus': [{
+                        'Name': preset.accelerator_name,
+                        'Manufacturer': preset.accelerator_manufacturer,
+                        'Count': preset.gpu,
+                        'MemoryInfo': {
+                            'SizeInMiB': preset.memory_gib * 1024 // preset.gpu
+                        },
+                    }],
+                    'TotalGpuMemoryInMiB': preset.memory_gib * 1024,
+                }
+                gpu_info = json.dumps(gpu_info_dict).replace('"', '\'')
+
+            writer.writerow({
+                'InstanceType': preset.fullname,
+                'AcceleratorName': preset.accelerator_name,
+                'AcceleratorCount': preset.gpu,
+                'vCPUs': preset.vcpu,
+                'MemoryGiB': preset.memory_gib,
```

Comment:
Hi @SalikovAlex, I was wondering, is `preset.memory_gib` the RAM of the machine or the GPU VRAM? Seeing that it's used in both `TotalGpuMemoryInMiB` and `MemoryGiB`.

One of our users raised an issue, saying the GPU VRAM of `gpu-h200-sxm_1gpu-16vcpu-200gb` is inconsistent with the actual number (141GB): https://github.com/skypilot-org/skypilot/issues/6622